packages/vite/src/node/plugins/reporter.ts

```diff
@@ -26,13 +26,14 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
   const compress = promisify(gzip)
   const chunkLimit = config.build.chunkSizeWarningLimit
 
-  const numberFormatter = new Intl.NumberFormat('en', {
+  const formatter = Intl.NumberFormat('en', {
+    style: 'unit',
+    unit: 'kilobyte',
+    notation: 'standard',
     maximumFractionDigits: 2,
     minimumFractionDigits: 2,
-  })
-  const displaySize = (bytes: number) => {
-    return `${numberFormatter.format(bytes / 1000)} kB`
-  }
+  }).format
+  const formatBytes = (bytes: number) => formatter(bytes / 1000)
```

Comment:
If we're only showing in `kB` and opting out of leveraging other units, it kinda begs the question why we need to pass in 3 more `NumberFormat` options when we can just append `kB` at the end and call it a day.

I'm also okay with scrapping this PR.

Reply:
I agree with you here. Let's keep the current one for now then. Thanks for sending the PR again, though, I think it was good to explore this option.